src/Extensions/Logging/Internal/OmexLoggerProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	internal class OmexLoggerProvider : ILoggerProvider, ISupportExternalScope
+	{
+		public OmexLoggerProvider(OmexLogsEventSource logsEventSource, IExternalScopeProvider defaultExternalScopeProvider)
+		{
+			m_logsEventSource = logsEventSource;
+			m_defaultExternalScopeProvider = defaultExternalScopeProvider;
+		}
+
+
+		public ILogger CreateLogger(string categoryName) => new OmexLogger(m_logsEventSource, m_externalScopeProvider ?? m_defaultExternalScopeProvider, categoryName);
+
+
+		public void Dispose() { }
```

Comment:
>	public void Dispose() { } [](start = 1, length = 26)

Do you need this method? Does one of the dependencies inherit from `IDisposable`?

Reply:
It's required by `ILoggerProvider` interface